src/mesh/mesh_smoother_vsmoother.C

```diff
@@ -140,7 +140,15 @@ void VariationalMeshSmoother::smooth(unsigned int)
   if (!_setup_called)
     setup();
 
-  system()->solve();
+  try
```

Comment:
Let's change these to libmesh_try and libmesh_catch for the `--disable-exceptions` builds?